benchmarks/utils/runbench.js

```diff
@@ -90,7 +90,8 @@ function runBenchmark (name, done) {
 }
 
 function sum (arr) {
-  var result = 0
+  let result = 0
+  /* eslint no-var: off */
```

Comment:
If we were to directly use `eslint` with https://github.com/standard/eslint-config-standard instead of the straight `standard` package, could we update our `.eslintrc` to include this rule? I really despise having these comments scattered all over the code just to tweak a development tool. I understand that sometimes they are unavoidable (e.g. `/* istanbul skip else */`), but the fewer we have to have the better.

In general, I'm really annoyed by the latest couple releases of `standard` imposing some really drastic rules. But this "no more var" one is asinine and dangerous.

Reply:
@jsumners tried now

`{
    "extends": "standard",
    "rules": {
        "no-var": "off"
    }
}`

and 

`{
    "extends": "standard",
    "rules": {
        "no-var": 0
    }
}`

But none is working, Am I missing something, did they change something in standard 16? Even tried .eslintrc.json

Reply:
Yes, we would have to remove the `standard` package and add the `eslint` package. Then update the `package.json` scripts accordingly.

Reply:
Done